### ISSUE TITLE
Image description/prodnote text color should be updated to increase contrast ratio against background (accessibility/WCAG violation)

### ISSUE THREAD

**Issue by martinpub** [mtmse/epub3-css#11]
Currently, ACE reports the following:

> o	Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
o	Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds
o	Elements must have sufficient color contrast

Fix this.
